cluster/config-defaults.yaml

```diff
@@ -342,10 +342,10 @@ skipper_open_policy_agent_styra_token: ""
 #   - disabled: scales controller to zero replicas
 #   - production: runs the controller
 #
-fabric_gateway_controller_mode: "disabled"
+fabric_gateway_controller_mode: "production"
```

Comment:
Let's drop all config items that are only to "enable" it.
https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/fabric-gateway/deployment.yaml#L11-L17

Reply:
I think this is nice to have, so we have the crd everywhere but can control the controller

Reply:
decided to also remove, and introduce  `fabric_gateway_controller_replicas` when we need